packages/server-wallet/src/db/migrations/20201014174039_objectives.ts

```diff
@@ -5,13 +5,18 @@ const tableName = 'objectives';
 export async function up(knex: Knex): Promise<any> {
   return knex.schema.createTable(tableName, function(table) {
     table.string('objective_id').primary();
-    table.string('status');
-    table.string('type');
+    table.string('status').notNullable();
+    table.string('type').notNullable();
     table.string('target_channel_id');
     table.string('funding_strategy');
```

Comment:
Should this be not nullable?

Reply:
no

Reply:
You want this to be null in the case of inserting an objective that does not have this property. 

Is your point that all objectives _should_ have this property? Currently, they do not all have this property.